src/main/java/org/usfirst/frc/team3786/robot/utils/LED.java

```diff
@@ -25,41 +20,41 @@ public void close() {
 		i2c.close();
 	}
 
-	public static void setRGB(int r, int g, int b) {
-		byte[] data = { brightness, (byte) r, (byte) g, (byte) b };
-		currentColor = data;
-		i2c.transaction(data, data.length, new byte[0], 0);
+	private static boolean send() {
+		int i = 0;
+		while (true) {
+			boolean abort = i2c.transaction(buffer, buffer.length, new byte[0], 0);

```

Comment:
Instead of allocating a new byte[0] with every iteration, create a static final byte[] = new byte[0], and use that.